test/mocha/10-api.js

```diff
@@ -153,4 +160,159 @@ describe('bedrock-account-http', function bedrockAccountHttp() {
     });
   });
 
+  describe('patch /:account', function() {
+    it('should update an account', async function() {
+      this.timeout(60000);
```

Comment:
Are you seeing issues with timeouts?  The 15s default works for most things, if it's not working then there may be some problem.

Reply:
good catch I was actually seeing a time out because I was not calling .send after setting status code